internal/naming/names.go

```diff
@@ -353,6 +353,15 @@ func ExporterWebConfigMap(cluster *v1beta1.PostgresCluster) metav1.ObjectMeta {
 	}
 }
 
+// OperatorConfigurationSecret returns the ObjectMeta necessary to lookup the
+// Secret containing PGO configuration.
+func OperatorConfigurationSecret() metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Namespace: config.PGONamespace(),
+		Name:      "pgo-config",
```

Comment:
Nothing `bridge` related here?

Reply:
Correct. We decided on this name with `bridge-` prefixes on the fields/keys in the Secret.